spyder/plugins/workingdirectory/confpage.py

```diff
@@ -23,32 +23,36 @@
 
 class WorkingDirectoryConfigPage(PluginConfigPage):
     def setup_page(self):
-        about_label = QLabel(_("The <b>current working directory</b> is "
-                    "the working directory for IPython consoles "
-                    "and the current directory for the File Explorer."))
+        about_label = QLabel(
+            _("This is the directory that will be set as the default for "
+              "the IPython console and Files panes.")
+        )
         about_label.setWordWrap(True)
 
+        # Startup directory
         startup_group = QGroupBox(_("Startup"))
         startup_bg = QButtonGroup(startup_group)
-        startup_label = QLabel(_("At startup, the current working "
-                                 "directory is:"))
+        startup_label = QLabel(
+            _("At startup, the working directory is:")
+        )
         startup_label.setWordWrap(True)
         lastdir_radio = self.create_radiobutton(
-                                _("The current project directory "
-                                  "or user home directory "
-                                  "(if no project is active)"),
-                                'startup/use_project_or_home_directory',
-                                True,
-                                _("At startup,"),  # TODO
-                                button_group=startup_bg)
+            _("The project or user home directory"),
+            'startup/use_project_or_home_directory',

```

Comment:
```suggestion
            _("The project (if open) or user home directory"),
            'startup/use_project_or_home_directory',
            tip=_("The startup working dir will be root of the current project"
                  "if one is open, otherwise the user home directory"),
```

I'm a bit concerned users may be confused by seeing that it can be either their project dir or their home dir, and not immediately understand when it is which. This tweak clarifies when the CWD is the home dir vs. the project dir, and provides more detail in the tooltip.